examples/playbooks/vmwscript/swarm.pb

```diff
@@ -0,0 +1,14 @@
+{{/* Runs the vmwscript playbook to provision a Swarm on VMSphere */}}
+{{/* =% vmwscript %= */}}
+
+{{ flag "vcenter-url" "string" "VCenter URL" | prompt "VCenter URL?" "string" "https://username@vsphere.local:password@vc.unifydc.io/sdk" | var "input/url"}}
+{{ flag "data-center" "string" "Data Center name" | prompt "Data Center Name?" "string" "Datacenter" | var "input/dc"}}
+{{ flag "data-store" "string" "Data Store name" | prompt "Data Store Name?" "string" "datastore1" | var "input/ds"}}
+{{ flag "network-name" "string" "Network name" | prompt "Network Name?" "string" "Internal Network (NAT)" | var "input/nn"}}
+{{ flag "vsphere-host" "string" "VSphere host" | prompt "Host Name?" "string" "exsi01.unifydc.io" | var "input/host"}}
```

Comment:
exsi01 => esxi01 .. That confused me why it wasn't working for a little while :-)